src/error.ts

```diff
@@ -64,11 +60,6 @@ export function GraphQLError(
   UpstreamGraphQLError.prototype,
   {
     constructor: { value: GraphQLError },
-    name: { value: "GraphQLError" },
-    toString: {
```

Comment:
Which part of the update requires removing `toString`?

Reply:
`printError` calls `toString` in v16 so this causes an infinite loop.
You can see the relevant change in v16;
https://github.com/graphql/graphql-js/pull/3252